### DISCUSSION TITLE
【官方文档错误】下拉选择器组件 select 文档问题

### DISCUSSION THREAD

**Discussion by crazyyf** [element-plus/element-plus#7576] [General]
【官方文档错误】下拉选择器组件 select 文档问题，Select 属性配置中，clearable（是否可以清空选项） 属性默认值应该是 true , 但是在文档中写的是 false ，在非 plus 版本同样存在此问题。

**Comment by btea** [comment]
文档是正确的，clearable为true时，鼠标移入输入框，会出现清空图标，如下图所示
![image](https://user-images.githubusercontent.com/24516654/168610111-69a29171-855f-45f8-a60e-d00872c33fc9.png)
